### PR TITLE
add i2c/pwm for radxa cm5 rpi cm4 io

### PR DESCRIPTION
**Pull request by Ken-Vamrs**

### CODE REVIEW

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
https://github.com/radxa-pkg/radxa-overlays/blob/main/arch/arm64/boot/dts/rockchip/overlays/rk3588-i2c6-m0.dts#L16

这里也麻烦改一下，SDA和SCL反了

**Comment by RadxaYuntian** [comment]
https://applink.feishu.cn/client/message/link/open?token=AmWMH9o6RAACZr8UNtMQwAQ%3D

**Comment by Ken-Vamrs** [comment]
> https://github.com/radxa-pkg/radxa-overlays/blob/main/arch/arm64/boot/dts/rockchip/overlays/rk3588-i2c6-m0.dts#L16
> 
> 这里也麻烦改一下，SDA和SCL反了

ok

File: arch/arm64/boot/dts/rockchip/overlays/rk3588-i2c6-m0.dts [hunk 1]
```diff
@@ -13,7 +13,7 @@
 On Radxa ROCK 5A this is SDA pin 27 and SCL pin 28.
 On Radxa ROCK 5C this is SDA pin 27 and SCL pin 28.
 On Radxa ROCK 5D this is SDA pin 27 and SCL pin 28.
-On Radxa CM5-RPI-CM4-IO this is SDA pin 3 and SCL pin 5.
+On Radxa CM5-RPI-CM4-IO this is SCL pin 3 and SDA pin 5.
```
Comment (RadxaYuntian) [inline arch/arm64/boot/dts/rockchip/overlays/rk3588-i2c6-m0.dts:16]:
```suggestion
On Radxa CM5-RPI-CM4-IO this is SDA pin 5 and SCL pin 3.
```

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**